packages/repository/src/__tests__/unit/repositories/relations-helpers/resolve-has-many-through-metadata.unit.ts

```diff
@@ -0,0 +1,275 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
```

Comment:
@agnes512, when your PR is almost ready to merge, could you please run the copyright command so that it generates the right header with the correct year? Thanks.